Backend/backend/src/test/java/com/example/backend/DogControllerTest.java

```diff
@@ -0,0 +1,7 @@
+package com.example.backend;
+
+public class DogControllerTest {
+
+    //Can't test DogController, it needs an add dog function
```

Comment:
Add tog sker via User

Reply:
Jag förstår inte, hittar inte något om DogController i User filen

Reply:
Jag syftatade på "it needs an add dog function". Den finns i user tror jag.